.test/meta-commands/out.sh

```diff
@@ -137,6 +137,29 @@ jq -s '
 	)
 ' temp/index.json > temp/index.json.new
 mv temp/index.json.new temp/index.json
+SOURCE_DATE_EPOCH=1709081058 \
+	docker buildx build --progress=plain \
+	--provenance=false \
+	--build-context "base:latest=oci-layout://$PWD/temp/@$(jq -r '.manifests[0].digest' index.json)" \
+	--sbom=generator="$BASHBREW_BUILDKIT_SBOM_GENERATOR" \
+	--output 'type=oci,tar=false,dest=temp_sbom/' \
+	- <<<'FROM base'
+newIndexDigest="$(jq -r '.manifests[0].digest | sub("sha256:";"")' temp_sbom/index.json)"
```

Comment:
I don't love the amount of sha256 assumptions in all this code (and calling things "digest" that aren't actually an OCI digest, but are just the raw checksum).  I _really_ wish there were more tools like `crane` for more easily querying/manipulating OCI layouts, but for now I think we can get away with a shell-ism like `"blobs/${foo/://}"` to get the path to our blobs given an OCI digest.